RELEASE.md

```diff
@@ -17,47 +17,49 @@ Steps for a HALE release
 ========================
 
 On release branch:
-
-1. Merge origin/master into release
+1. Merge origin/master into release and commit the changes
 2. Set the release version for application bundles and build
    `./updateversionnumbers.groovy --release <RELEASE-VERSION>`
 3. Create a commit
 4. Create a tag for the new release
 5. Push to origin/release and push tags
+6. Create a PR
```

Comment:
If we do this with PRs now, this step should replace step nr. 5, I think.

Reply:
As I've done it, I have first pushed to my `local release branch/another branch starting from release` and just after I have created a PR. 
Ah , I see origin/release should be replaced by something like `local_branch` or something similar

Reply:
@emanuelaepure10 I'll remove the `origin/` than

Reply:
No, as I said before: either we push directly to the `origin` branch or we create a branch and a PR to the origin branch. If we go with the PR approach, step 5 must not happen.

Reply:
But you need to push to your local branch...otherwise on what the Pr can be created?